From 92566cb41e90a4ba432c2e72ee3751bd41d47e0d Mon Sep 17 00:00:00 2001
From: Rakesh Kudurumalla <rkudurumalla@marvell.com>
Date: Tue, 21 Mar 2023 15:09:18 +0530
Subject: [PATCH 330/955] ci: reduce l3fwd reference numbers

due to change in toolchain from gcc10 to
gcc12 there is a slight impact on performance
so reduce reference numbers to make l3fwd app
to pass ci

Signed-off-by: Rakesh Kudurumalla <rkudurumalla@marvell.com>
Change-Id: I0b080f760455c922a803cae28af5c77adabfeb25
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/99601
Base-Builds: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Reviewed-by: Nithin Kumar Dabilpuram <ndabilpuram@marvell.com>
---
 .../fwd_perf/ref_numbers/rclk2200_sclk1100.cn96.l3fwd           | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/marvell-ci/test/cnxk-tests/fwd_perf/ref_numbers/rclk2200_sclk1100.cn96.l3fwd b/marvell-ci/test/cnxk-tests/fwd_perf/ref_numbers/rclk2200_sclk1100.cn96.l3fwd
index ba24be046cf65..0a275bc414295 100644
--- a/marvell-ci/test/cnxk-tests/fwd_perf/ref_numbers/rclk2200_sclk1100.cn96.l3fwd
+++ b/marvell-ci/test/cnxk-tests/fwd_perf/ref_numbers/rclk2200_sclk1100.cn96.l3fwd
@@ -1,4 +1,4 @@
 PPS of tests on 96xx and multiple RCLK
 ---------------------------------------
 
-L3FWD_1C 25300000
+L3FWD_1C 24800000
-- 
2.25.1

